app-resources/src/main/resources/json/views/paikkis-3D.json

```diff
@@ -0,0 +1,224 @@
+    },
+    { "id": "drawtools" },
+    { "id" : "divmanazer" },
+    { "id": "toolbar" },
+    { "id" : "statehandler" },
+    {
+      "id": "infobox",
+      "config": {
+        "adaptable": true
+      }
+    },
+    { "id" : "search" },
+    {
+      "id" : "layerselector2",
+      "config": {
+        "showSearchSuggestions": true
+      }
+    },
+    { "id" : "layerselection2" },
+    { "id" : "maplegend" },
+    { "id" : "publisher2" },
+    { "id" : "statsgrid" },
+    { "id" : "analyse" },
+    { "id" : "metadataflyout" },
+    {
+      "id" : "userguide",
+      "config": {
+        "tabs": [
+          {
+            "title": "Karttaikkuna",
+            "tags": "ohje_karttaikkuna"
+          },
+          {
+            "title": "Kartan julkaisu",
+            "tags": "ohje_karttajulkaisu"
+          },
+          {
+            "title": "Teemakartat",
+            "tags": "ohje_teemakartat"
+          },
+          {
+            "title": "Analyysi",
+            "tags": "ohje_analyysi"
+          }
+        ]
+      }
+    },
+    {
+      "id" : "personaldata",
+      "config": {
+        "publishedMapUrl": {
+          "en": "/published/en/",
+          "fi": "/published/fi/",
+          "sv": "/published/sv/"
+        }
+      }
+    },
+    { "id" : "myplaces3" },
+    { "id" : "guidedtour" },
+    { "id" : "backendstatus" },
+    { "id" : "printout" },
+    {
+      "id" : "featuredata2",
+      "config": {
+        "selectionTools": true,
+        "singleSelection" : true
+      }
+    },
+    { "id" : "metadatacatalogue" },
+    { "id" : "myplacesimport" },
+    { "id" : "findbycoordinates" },
+    { "id" : "heatmap" },
+    {
+      "id" : "coordinatetool",
+      "config": {
+        "isReverseGeocode": true,
+        "reverseGeocodingIds": "WHAT3WORDS_CHANNEL",
+        "supportedProjections": [
+          "EPSG:3067",
+          "NLSFI:etrs_gk",
+          "NLSFI:ykj",
+          "EPSG:4258",
+          "LATLON:kkj",
+          "EPSG:3046",
+          "EPSG:3048",
+          "EPSG:3873",
+          "EPSG:3874",
+          "EPSG:3875",
+          "EPSG:3876",
+          "EPSG:3877",
+          "EPSG:3878",
+          "EPSG:3879",
+          "EPSG:3880",
+          "EPSG:3881",
+          "EPSG:3882",
+          "EPSG:3883",
+          "EPSG:3884",
+          "EPSG:3885"
+        ],
+        "showEmergencyCallMessage": true,
+        "projectionShowFormat": {
+          "decimals": 3,
+          "format": "metric",
+          "EPSG:4258": {
+            "decimals": 3,
+            "format": "degrees"
+          },
+          "LATLON:kkj": {
+            "decimals": 3,
+            "format": "degrees"
+          },
+          "EPSG:3067": {
+            "decimals": 0
+          }
+        }
+      }
+    },
+    { "id" : "timeseries" },
+    {
+      "id" : "feedbackService",
+      "config": {
+        "title": "Open 311 service",
+        "bundleinstancename": "feedbackService",
+        "bundlename": "feedbackService"
```

Comment:
This looks a bit whacky. Is this what the bundle actually needs?

Reply:
Nothing documented about these here either http://www.oskari.org/api/bundles#unreleased/mapping/feedbackService